include/boost/math/quadrature/trapezoidal.hpp

```diff
@@ -111,22 +111,15 @@ auto trapezoidal(F f, Real a, Real b, Real tol, std::size_t max_refinements, Rea
 
     return static_cast<K>(I1);
 }
-#if BOOST_WORKAROUND(BOOST_MSVC, < 1800)
+#if defined(_MSC_VER) && (_MSC_VER < 1800)
```

Comment:
This was a workaround for vc-11 and earlier: that compiler was not C++11 compliant or anywhere near it and won't be supported any more, so this whole #if branch can go.